semantic_release/hvcs/bitbucket.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         hvcs_api_domain: str | None = None,
         token: str | None = None,
         allow_insecure: bool = False,
-        **kwargs: Any,
+        **kwargs: Any,  # noqa: ARG002
```

Comment:
I'm not sure - seems like we maybe don't need kwargs on either the parent or child for these? But not sure if I'm missing some implicit way they're used?

Reply:
Its for the next improvement. It doesn't have much value currently but it will when different HVCS constructors have different parameters.  You must use the `**kwargs` argument to consume any extra args that are not recognized and gracefully ignore them especially because the constructor signature will be slightly different.